src/tests/test_verify_schema.sh

```diff
@@ -1,23 +1,44 @@
 #!/bin/bash -e
 
-PYTHONPATH="$PYTHONPATH:$TRAVIS_BUILD_DIR/src"
+# Use the Travis build dir if specified, otherwise assume that the
+# repo is unpacked in the current working dir
+REPO_ROOT_DIR=${TRAVIS_BUILD_DIR:-.}
+
+PYTHONPATH="$PYTHONPATH:$REPO_ROOT_DIR/src"
 export PYTHONPATH
 
 function verify_xml {
     # Verifies XML data against the schema
     xml="$1"
     type="$2"
     echo "Validating $type XML schema..."
-    xmllint --noout --schema "$TRAVIS_BUILD_DIR/src/schema/$type.xsd" $xml
+    xmllint --noout --schema "$REPO_ROOT_DIR/src/schema/$type.xsd" $xml
 }
 
-if [[ "$TRAVIS_PULL_REQUEST" == "false" ]]; then
+if [[ $TRAVIS_PULL_REQUEST == "false" ||
+      $GH_EVENT == 'push' &&
+      $GITHUB_REPOSITORY == 'opensciencegrid/topology' ]]; then
+    # Ensure that the .ssh dir exists
+    mkdir ~/.ssh
+    chmod 0700 ~/.ssh
     cat >> ~/.ssh/known_hosts <<EOF
 bitbucket.org ssh-rsa AAAAB3NzaC1yc2EAAAABIwAAAQEAubiN81eDcafrgMeLzaFPsw2kNvEcqTKl/VqLat/MaB33pZy0y3rJZtnqwR2qOOvbwKZYKiEO1O6VqNEBxKvJJelCq0dTXWT5pbO2gDXC6h6QDXCaHo6pOHGPUy+YBaGQRGuSusMEASYiWunYN0vCAI8QaXnWMXNMdFP3jHAJH0eDsoiGnLPBlBp4TNm6rYI74nMzgz3B9IikW4WVK+dc8KZJZWYjAuORU3jc1c/NPskD2ASinf8v3xnfXeukU0sJ5N6m5E8VLjObPEO+mN2t/FZTMZLiFqPWc/ALSqnMnnhwrNi2rbfg/rd/IpL8Le3pSBne8+seeFVBoGqzHM9yXw==
 EOF
 
-    openssl aes-256-cbc -K $encrypted_457175ef53a3_key -iv $encrypted_457175ef53a3_iv -in src/tests/contacts.enc -out contacts -d
+    touch contacts
     chmod 600 contacts
+
+    if [[ $GH_EVENT == 'push' ]]; then
+        echo "$CONTACT_DB_KEY" > contacts
+    else
+        openssl aes-256-cbc \
+                -K $encrypted_457175ef53a3_key \
+                -iv $encrypted_457175ef53a3_iv \
```

Comment:
These variables are created by Travis (for the secrets). Does GHA do the same thing?

Reply:
Not quite. For GHA, you can store the secrets in GitHub and access them directly (https://docs.github.com/en/free-pro-team@latest/actions/reference/encrypted-secrets):

- In the jobs and steps
- In scripts as long as you specify the env in the step (https://github.com/opensciencegrid/topology/pull/1441/files#diff-c4078cc3c976edd304cd31c0b08976d09c6af057bad110660e18ec1b856d8db1R23 and https://github.com/opensciencegrid/topology/pull/1441/files#diff-051eaf796f103706c605e10ddc25ecc7f628058cd6c52c7f362ec90d4ca537f1R32)

Reply:
I see. Also, I just noticed that this line only runs for travis builds so I think you're good.